vitess-mixin/e2e/docker-compose.beginners.yml

```diff
@@ -90,16 +90,16 @@ services:
       - "15306:$MYSQL_PORT"
     command: ["sh", "-c", "/vt/bin/vtgate \
         $TOPOLOGY_FLAGS \
-        -logtostderr=true \
-        -port $WEB_PORT \
-        -grpc_port $GRPC_PORT \
-        -mysql_server_port $MYSQL_PORT \
-        -mysql_auth_server_impl none \
-        -cell $CELL \
-        -cells_to_watch $CELL \
-        -tablet_types_to_wait PRIMARY,REPLICA \
-        -service_map 'grpc-vtgateservice' \
-        -enable_system_settings=true \
+        --logtostderr=true \
+        --port $WEB_PORT \
+        --grpc_port $GRPC_PORT \
```

Comment:
There are other docker-compose file too and vtcompose in this directory as well. Are they already up to date? I remember making a change but I am not sure.

Reply:
Just checked, the PR was https://github.com/vitessio/vitess/pull/11188/ and it seems I made the other changes, just this one slipped. Thank-you for catching this 💯

Reply:
Oh! I see now, this is the vitess-mixin directory. Apparently we have 2 vtcompose.go files! 😖 
one here and one in examples/compose

Reply:
this is the ones I was able to get through finder